connector/connect/server/pom.xml

```diff
@@ -113,6 +113,12 @@
       <version>${project.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-connect-client-jvm-shaded_${scala.binary.version}</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
instead of shading, if I could somehow define in the server that I want to filter spark-connect-client-jvm, and include classes from org.apache.spark.sql.connect, but exclude the rest that could also maybe work...